pkg/cluster/config/validate.go

```diff
@@ -30,7 +30,7 @@ func (c *Config) Validate() error {
 	// All nodes in the config should be valid
 	for i, n := range c.Nodes {
 		if err := n.Validate(); err != nil {
-			errs = append(errs, fmt.Errorf("please fix invalid configuration for node %d: \n%v", i, err))
+			errs = append(errs, errors.Wrapf(err, "invalid configuration for node %d", i))
```

Comment:
yeah unfortunately this error is very likely to be used for """display""" in the terminal. we should probably circle back to this later.

Reply:
thanks, I noticed this one, I thought they can wrap without line breaks, no?

Reply:
IIRC the formatting for the underlying err has line breaks as well, I think wrapf might not handle this well.

Reply:
Hmm, to be clear, what I thought was the texts of the error message would wrap themselves in the terminal, so I removed the `\n` line break. Otherwise, we would have a second line start with `: `, right?

Could you confirm what we need to do here? I'm happy to modify or revert this.

Reply:
yes, also i think `\n` is the `error` strings is not a good idea.
instead here should be a custom error type passed around and it should include a field that should be printed optionally by consumers of the error.

Reply:
right, preferably a smarter aggregate type that had a pretty format mode that could be accessed (similar to the stacktrace from this package) would be ideal. unfortunately this package only seems to do stack trace type aggregates, but we need multi-error too

Reply:
`errors.Wrapf()` or `errors.Errorf()` here?

Reply:
I think errorf, let's just get everything onto this package at least, and revamp the actual error contents in another PR 😅

Reply:
` errors.Wrapf(err, "invalid configuration for node %d", i))`
is/was correct here.